.travis.yml

```diff
@@ -94,9 +94,11 @@ script:
 
   # FVP
   - $make PLATFORM=vexpress-fvp CFG_ARM32_core=y
+  - $make PLATFORM=vexpress-fvp CFG_ARM32_core=y CFG_PL061=y CFG_PL022=y
```

Comment:
I'm wondering if we should enable PL061 and PL022 by default on HiKey instead. Then, no need to add anything to `.travis.yml` to have the code build-tested by Travis. Enabling them on other platforms would not check more things it seems. What do you think?